README.md

```diff
@@ -109,13 +100,33 @@ Finished: SUCCESS
 The exit code reflects the result of the build. The `test` directory of this workspace includes a very simple
 example of Jenkinsfile that can be used to demo Jenkinsfile Runner.
 
-### Passing parameters
+### CLI options
+The executable of Jenkinsfile Runner allows its invocation with these cli options:
 
-Any parameter values, for parameters defined on workflow job within `parameters` statement
-can be passed to the Jenkinsfile Runner using `-a` or `--arg` switches in key=value format. 
+```
+ # Usage: jenkinsfile-runner
+ --runWorkspace FILE     : Path to the workspace of the run to be used within
+                           the node{} context. It applies to both Jenkins
+                           master and agents (or side containers) if any.
+                           Requires Jenkins 2.119 or above
+ -a (--arg)              : Parameters to be passed to workflow job. Use
+                           multiple -a switches for multiple params
+ -f (--file) FILE        : Path to Jenkinsfile (or directory containing a
+                           Jenkinsfile) to run, default to ./Jenkinsfile.
+ -ns (--no-sandbox)      : Disable workflow job execution within sandbox
+                           environment
+ -p (--plugins) FILE     : plugins required to run pipeline. Either a
+                           plugins.txt file or a /plugins installation
+                           directory. Defaults to plugins.txt.
+ -v (--version) VAL      : jenkins version to use. Defaults to latest LTS.
+ -w (--jenkins-war) FILE : path to jenkins.war or exploded jenkins war directory
+```
 
-Passing parameters defined within `parameters` section of the pipeline is optional. 
+where `--runWorkspace`, `-ns` and `-a` are optional.
```

Comment:
Add it to the text instead? Otherwise it might be difficut to maintain the diffs

Reply:
It is also already incorrect, because `-v` is optional

Reply:
Fixed!